.ctrc.yml

```diff
@@ -15,10 +14,10 @@ hooks:
         echo "$(conventional-tools commitgen)$(cat ${1})" > ${1};
       fi
   pre-commit:
-    - $(git rev-parse --show-toplevel)/.gitlab/hooks/check-email.sh
-    - $(git rev-parse --show-toplevel)/.gitlab/hooks/check-copyright.sh
-    - $(git rev-parse --show-toplevel)/.gitlab/hooks/no-yaml-files.sh
-    - $(git rev-parse --show-toplevel)/.gitlab/hooks/prettier.sh
+    - $(git rev-parse --show-toplevel)/.github/hooks/check-email.sh
```

Comment:
I think this could need another ticket but, this should really be removed, not. I will need to remove it as I do not have a practically email any more.

Reply:
Yep I agree, it can be removed